index.js

```diff
@@ -96,12 +97,12 @@ GitHub.prototype = {
    * @api public
    */
 
-  getAll: function(path, cb) {
-    var opts = this.defaults('GET', path);
-    opts.path = this.interpolate(opts);
-    gh.requestAll(opts, cb);
-    return this;
-  },
```

Comment:
how will this be handled now? this was the only reason I was using `github-request`. It's not hard to do the paging, but we need to be able to handle requests for 100s/1000s of records. 

I can explain how to do this if you're not familiar with it...